### ISSUE TITLE
Not able to interact with the majority of the UI

### ISSUE THREAD

**Issue by erbrinks** [Chatterino/chatterino2#2322]
**Describe the bug**
Not able to click on anything besides the tabs to select the channel and delete that channel. (Not able to close the application, click settings, login, minimize, or click to maximize).

Able to add splits but I cannot interact with the UI that pops up to specify the split. However, I can remove splits and interact with the prompt that pops up asking for confirmation.

As far as I know, I am able to type in the chat if I could login due to the fact that I can type in the text box.

In terms of looks, everything is zoomed in and the text is blurry. Saw something similar to that visually when I was looking for solutions but did not see a fix for it. Emotes are working fine as far as I can tell.

**To reproduce**
Launching the application (administrator mode or not).

**Screenshots**
![image](https://user-images.githubusercontent.com/56411406/103369802-118f7c80-4a99-11eb-9633-ceb1add0b9d0.png)

**Chatterino version**
Tried version 2.2.2-fix and 2.2.4-beta2. The application worked awhile ago and I was using it all the time but I am not sure of the version that I was using. It broke when I was using that version so I am not sure it would make a difference anyway.

**Operating system**
Windows 10 (Latest updates)

**Additional information**
Noticed the issue after I ran the application on my vertical monitor awhile ago. Not sure if this is related in any way. Does not work on either monitor now.

Main monitor: 3840 x 2160
Vertical monitor: 1080 x 1920

**Comment by alazymeme** [comment]
Sounds like a DPI issue. What DPI are both monitors set to? Chatterino works just fine vertically for me:

![image](https://user-images.githubusercontent.com/12804673/103397381-dfa71480-4b83-11eb-8f84-1ef327b49c9c.png)

**Comment by erbrinks** [comment]
I went in and changed the scaling from 150% on my main monitor to 100% and it worked on both monitors. I changed it back without closing the program and it kept working. However, if I keep it at 150% and close the app then re-open it, the same issue arises. 

It's not a really big deal, but it would be annoying to have to change the scaling from 150% to 100% and back again when I want to use the app. 

If this is something that is not on your end and cannot be patched I totally understand. Thanks for the input and quick response though!
